fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/plan/temp/CascadesPlanner.java

```diff
@@ -237,6 +244,15 @@ public RecordQueryPlan plan(@Nonnull RecordQuery query) {
         }
     }
 
+    @Nonnull
+    @Override
+    public QueryPlanResult planForQuery(@Nonnull final RecordQuery query) {
```

Comment:
This method's name is not really reflecting what's going on. `planWithInfo` maybe?

Reply:
Done.